components/routing.rst

```diff
@@ -41,7 +41,7 @@ your autoloader to load the Routing component::
 
     $context = new RequestContext('/');
```

Comment:
here would be $requestContext too

Reply:
Ah, I started renaming `context` to `requestContext` but then I saw `context` is used everywhere, not sure if I should rename all or revert this one.